python/lsst/afw/geom/testUtils.py

```diff
@@ -857,17 +858,10 @@ def checkTransformFromFrameSet(self, fromName, toName):
 
             self.checkPersistence(transform)
 
-            frameSetCopy = transform.getFrameSet()
-            transformCopy = TransformClass(frameSetCopy)
+            mappingFromFrameSet = transform.getMapping()
```

Comment:
Why is this one called `mappingFromFrameSet`?

Reply:
Good catch. I'll fix it.